### ISSUE TITLE
Pycharm plugin + Poetry env - No module named reloadium

### ISSUE THREAD

**Issue by benyaming** [reloadware/reloadium#80]
## Describe the bug
It looks like pycharm plugin installs the library to the system interpreter, even if there is a project-related virtual env.

Here is the pycharm output:
```
/Users/benyaming/Library/Caches/pypoetry/virtualenvs/...-py3.10/bin/python -m reloadium run /Users/benyaming/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/223.7571.203/PyCharm.app/Contents/plugins/python/helpers/pycharm/pycharm_load_entry_point.py /Users/benyaming/PycharmProjects/.../local_prep.ini 

> /Users/benyaming/Library/Caches/pypoetry/virtualenvs/...-py3.10/bin/python: No module named reloadium
```

## To Reproduce
N/A

## Expected behavior
Start the application

## Screenshots
N/A

## Desktop (please complete the following information):**
 - OS: MacOS
 - OS version: 12.5.1
 - M1 chip: no
 - Reloadium package version: n/a (didn't find where it was installed to)
 - PyCharm plugin version: 0.9.1
 - Editor: PyCharm 2022.3
 - Python Version: 3.10.6
 - Python Architecture: 64
 - Run mode: Run

## Additional context
Python 3.10 (managed by Pyenv) + poetry env

Not sure, if it is possible to run a file via local poetry env AND global reloadium?

Anyway, thanks a lot for the great project!

**Comment by dkrystki** [comment]
Hi @benyaming 
Does this directory exist on your computer `/usr/libexec/rosetta/` ?

Also could you follow these steps https://github.com/reloadware/reloadium/issues/57#issuecomment-1304731975 to debug this?

**Comment by benyaming** [comment]
@dkrystki,
> Does this directory exist on your computer /usr/libexec/rosetta/ ?  

Yes, it does


Regarding the debug -- the main issue is that the plugin can't find any version of the library

As I said above, I manually installed the library via local pip, and it works now, but the library is located at my env's `site-packages`. I added the lines there, here is the output (I added newlines manually):
```
[
'/', 
'/Users/benyaming/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/223.7571.203/PyCharm.app/Contents/plugins/python/helpers/pycharm', 
'/Users/benyaming/.reloadium/package/3.7', 
'/Users/benyaming/PycharmProjects/appcardapi', 
'/Users/benyaming/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/223.7571.203/PyCharm.app/Contents/plugins/python/helpers/pycharm_display', 
'/Users/benyaming/.pyenv/versions/3.10.6/lib/python310.zip', 
'/Users/benyaming/.pyenv/versions/3.10.6/lib/python3.10', 
'/Users/benyaming/.pyenv/versions/3.10.6/lib/python3.10/lib-dynload', 
'/Users/benyaming/Library/Caches/pypoetry/virtualenvs/appcardapi-BU645V_v-py3.10/lib/python3.10/site-packages', 
'/Users/benyaming/Library/Application Support/JetBrains/Toolbox/apps/PyCharm-P/ch-0/223.7571.203/PyCharm.app/Contents/plugins/python/helpers/pycharm_matplotlib_backend'
]

```

Also,
```
> ls -la /Users/benyaming/.reloadium/package
drwxr-xr-x  5 benyaming  staff  160 Dec  7 12:17 .
drwxr-xr-x  5 benyaming  staff  160 Dec  7 14:04 ..
drwxr-xr-x  4 benyaming  staff  128 Dec  7 12:17 3.10
drwxr-xr-x  4 benyaming  staff  128 Dec  7 12:17 3.9
-rw-r--r--  1 benyaming  staff    5 Dec  7 12:17 version.txt
```

Not sure why we have 3.7 in the path, but the actual versions are missing

**Comment by dkrystki** [comment]
Existence of `/usr/libexec/rosetta/` determines whether reloadium is run on M1 machine or not. 
It's not the most elegant way but other's didn't work at all unfortunately. 

I don't fully understand why this directory exists on some computers since rosetta shouldn't be needed on non M1 machines.
Could you try deleting it and reinstalling the plugin?

**Comment by benyaming** [comment]
@dkrystki may be a dumb question, but what is wrong with these ways?

https://superuser.com/questions/148500/determine-the-architecture-of-a-mac-from-the-command-line-or-script

**Comment by benyaming** [comment]
And I can not remove the rosetta folder, even under sudo. All guides are for M1, and they started with 
`pkgutil --files com.apple.pkg.RosettaUpdateAuto`, but on my MacBook, it returns `No receipt for 'com.apple.pkg.RosettaUpdateAuto' found at '/'.`

**Comment by dkrystki** [comment]
Fixed in reloadium 0.9.7 and PyCharm plugin 0.9.2
